adapters/goal.ts

```diff
@@ -11,8 +11,10 @@ const goalSiteConfig = {
 }
 
 export const fetchGoals = async (): Promise<Goal[]> => {
+  const config = getConfig();
+
   const goals: Goal[] = await axios
-    .get(`${API.GOAL_BASE_URL}/v1/goal/`)
+    .get(`${config.GOALS_API}/v1/goal/`)
```

Comment:
This URL naming is supposed to be the same as the URL either singular or plural depending on the incoming data.
Also verify API naming convention.

Reply:
I couldn't understand you. Do you mean it should be `.../v1/goals`?

Reply:
Yes

Reply:
I don't think it is dependent on that unless I'm understand it wrong